js/src/ui/TxHash/txHash.js

```diff
@@ -89,14 +95,14 @@ class TxHash extends Component {
       const num = blockNumber.minus(transaction.blockNumber).plus(1);
       txBlock = `#${transaction.blockNumber.toFormat(0)}`;
       confirmations = num.toFormat(0);
-      value = num.gt(10) ? 10 : num.toNumber();
+      value = num.gt(maxConfirmations) ? maxConfirmations : num.toNumber();
     }
 
     return (
       <div className={ styles.confirm }>
         <LinearProgress
           className={ styles.progressbar }
-          min={ 0 } max={ 10 } value={ value }
+          min={ 0 } max={ maxConfirmations } value={ value }
```

Comment:
attribute per line (inconsistency here, I know not introduced here, but since we are fixing it, readability counts.)